spec/javascripts/app/utils/events_spec.js

```diff
@@ -0,0 +1,89 @@
+const Events = require('../../../../app/assets/javascripts/app/utils/events').default;
+/*eslint-disable */
+const sinon = require('sinon');
```

Comment:
Yeah, I think we definitely want to be unit testing our javascript where possible. Self contained classes that don't have explicit dependencies on the DOM or browser APIs are great candidates for that.